trie/verkle.go

```diff
@@ -217,22 +217,16 @@ func (t *VerkleTrie) Hash() common.Hash {
 }
 
 // Commit writes all nodes to the tree's memory database.
-func (t *VerkleTrie) Commit(_ bool) (common.Hash, *trienode.NodeSet, error) {
-	root, ok := t.root.(*verkle.InternalNode)
-	if !ok {
-		return common.Hash{}, nil, errors.New("unexpected root node type")
-	}
-	nodes, err := root.BatchSerialize()
-	if err != nil {
-		return common.Hash{}, nil, fmt.Errorf("serializing tree nodes: %s", err)
-	}
+func (t *VerkleTrie) Commit(_ bool) (common.Hash, *trienode.NodeSet) {
+	root := t.root.(*verkle.InternalNode)
```

Comment:
Can we be sure that the type of this root node must be an internal node?
IIUC, it could also be EmptyNode if the trie is not initialized yet?

Reply:
We don't have the same check in MPT.  Is there a reason that we would be calling `Commit` on a non-initialized vkt Trie?

Reply:
The verkle trie will be initialized with one one internal node by default: https://github.com/ethereum/go-verkle/blob/master/tree.go#L250